src/store.ts

```diff
@@ -1,24 +1,39 @@
 import raygun from "raygun4js";
-import { configureStore, Action, getDefaultMiddleware } from "@reduxjs/toolkit";
+import { configureStore, Action, getDefaultMiddleware, Middleware, Dispatch } from "@reduxjs/toolkit";
 import rootReducer, { RootState } from "./rootReducer";
 import { ThunkAction } from "redux-thunk";
 import {
   mixerMiddleware,
   mixerKeyboardShortcutsMiddleware,
   startNewsTimer,
 } from "./mixer/state";
-import { persistStore } from "redux-persist";
+import {
+  persistStore,
+  FLUSH,
+  REHYDRATE,
+  PAUSE,
+  PERSIST,
+  PURGE,
+  REGISTER
+} from "redux-persist";
+
+const raygunMiddleware: Middleware < {}, RootState, Dispatch < any >> = (store) => (next) => (action) => {
```

Comment:
Did Prettier make the angle brackets like this? Because this is WordPress code style, which I despise.

Reply:
The GitHub Action says no :p

Reply:
I think I need to add auto-prettier to a VS Code plugin :P